stock_prices/stock_prices.py

```diff
@@ -3,7 +3,13 @@
 import argparse
 
 def find_max_profit(prices):
-  pass
+  max_prof = prices[1] - prices[0]
+  for i in range(1, len(prices)):
+    for j in range(i+1, len(prices)):
+      diff = prices[j] - prices[i]
+      if diff > max_prof:
+        max_prof = diff
+  return max_prof
```

Comment:
Hey Charlie, this is a nice solution. It's simple and easy to understand, which is always great for others looking at your code 😁 
There are a couple of loops in here that you can refactor out to improve efficiency, any ideas how to do that?